src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -61,10 +138,54 @@ type internal Return =
         Semicolon: Terminal
     }
 
+/// The kind of qubit declaration.
+type internal QubitDeclarationKind =
+
+    /// <summary>
+    /// Indicates a <c>use</c> qubit declaration.
+    /// </summary>
+    | Use
+
+    /// <summary>
+    /// Indicates a <c>borrow</c> qubit declaration.
+    /// </summary>
+    | Borrow
+
+/// The concluding section of a qubit declaration.
+type internal QubitDeclarationCoda =
+
+    /// The semicolon.
+    | Semicolon of Terminal
+
+    /// The block of statements after the declaration.
+    | Block of Statement Block
+
+/// A qubit declaration statement.
+and internal QubitDeclaration =
+    {
+        /// The kind of qubit declaration.
+        Kind: QubitDeclarationKind
+
+        /// The keyword used in the declaration.
+        Keyword: Terminal
+
+        /// The qubit binding.
+        Binding: QubitBinding
+
+        /// Optional open parentheses.
+        OpenParen: Terminal option
+
+        /// Optional close parentheses.
+        CloseParen: Terminal option
```

Comment:
nit: It's slightly confusing to me that `OpenParen` is after `Binding` in the record fields, since the `(` actually occurs before the binding in the syntax.

Reply:
moved the OpenParen to before the Binding